deprecated/packages/transifex-client/transifex-client.nuspec

```diff
@@ -18,18 +18,11 @@
     <tags>transifex-client transifex translation cli cross-platform foss</tags>
     <summary>The Transifex Command-line Tool enables you to manage your translations within a project without the need of an elaborate UI system.</summary>
     <!-- Do not touch the description here in the nuspec file. Description is imported during update from the Readme.md file -->
-    <description><![CDATA[The Transifex Command-line Tool enables you to manage your translations within a project without the need of an elaborate UI system.
-
-You can use the command line tool to create new resources, map locale files to translations, and synchronize your Transifex project with your local repository. Translators and localization managers can use it to handle large volumes of translation files. The Transifex Command-line Tool can help to enable continuous integration workflows and can be run from CI servers like Jenkins and Bamboo.
-
-# Notes
-- This package uses the Python 3.x version of the transifex-client
-- This package only provides the 64-bit executable due to a number of positives on VirusTotal.
-  The Software Authors believe them to be false positives, but I as the package maintainer am not convinced.
-]]></description>
+    <description><![CDATA[This package is deprecated as the upstream maintainer has deprecated the Python verson of the Transifex client along with the API endpoints it used.
+    Please use the replacement transifex-cli package going forward.]]></description>
     <releaseNotes>https://github.com/transifex/transifex-client/releases/tag/0.14.4</releaseNotes>
```

Comment:
Suggestion based on what we have done when previously deprecating a package.

```suggestion
    <releaseNotes>#### New users

* Please don't install this package. Install **transifex-cli** instead.

#### Existing users - Updating

* Updating this package will automatically install the most recent version of the **transifex-cli** chocolatey package.
* Please update any links, dependencies, or scripts that reference this package to the new package name **transifex-cli**

#### Existing users - Post-update Cleanup (optional)

`choco uninstall transifex-cli --whatif`</releaseNotes>
```

Reply:
I am honestly not sure if this should be as part of the release notes, or the description. I am not too fuzzed where it is added, but either in the description or in the release notes.

Reply:
Updated, thanks!